tailwind.config.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
         'app-background': '#fff',
         orange: '#ff5a42',
       },
+      fontFamily: {
+        // prettier-ignore
+        paragraph: ['Raleway','sans-serif'],
+      },
       fontSize: {
         fb: '1.188rem',
```

Comment:
why is font size defined for fb differently? Where is this used?

Reply:
@priyankajayaswal added this